my-app/src/Components/Main/Main.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import WeatherSearch from '../WeatherSearch/WeatherSearch';
+
+export default function Main(props) {
+  return (
+    <>
```

Comment:
` <>` empty tags are not needed if you have a common tag that wraps all the markup of the component